assets/src/blocks/ENForm/FormGenerator.js

```diff
@@ -250,22 +247,22 @@ const RadioInput = ({field, onInputChange, onBlur, errors}) => {
       </span><br />
       { inputs }
       {has_error &&
-        <div className="invalid-feedback">{ errors[field.id] ?? errorMessage }</div>
+        <div className="invalid-feedback">{ errors[ field.id ] ?? errors.errorMessage }</div>
```

Comment:
For the other field types we define an `errorMessage` string, we don't get it from the `errors` object so I think we should do the same here?